src/index.js

```diff
@@ -0,0 +1,126 @@
+import './style.css';
+import getDataFromLocalStorage from './modules/localStorage.js';
+import updateView from './modules/updateView.js';
+
+updateView();
+
+const storeItem = (items) => {
+  if (items.length > 0) {
+    localStorage.setItem('todoList', JSON.stringify(items));
+  } else {
+    localStorage.clear();
+  }
+};
+
+const clearLocalStorage = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/38088818/166902527-e6fbc563-8576-48d8-81c6-ad5bea17104c.png)

[OPTIONAL] When I click the recycle icon, all to-do items should be cleared from storage whether they are checked or unchecked. Currently noting is happening, kindly add the functionality for clearing all items when the recycle icon is clicked.